src/module.c

```diff
@@ -299,6 +299,7 @@ NAPI_METHOD(dcn_context_unref) {
   NAPI_DCN_CONTEXT();
 
   TRACE("Unrefing dc_context");
+  dcn_context->gc = 1;
```

Comment:
Can you describe why this fixed the problem? ^^

Reply:
because the event loop checks for it and you forgot to put it back while you were refactoring it a long time ago.

Reply:
line 176 if you wonder